style.css

```diff
@@ -60,7 +59,7 @@ body {
   display: flex;
   background: url(./images/background1.png);
   justify-content: center;
-  background-size: 100% 100%;
+  background-size: cover;
   background-repeat: no-repeat;
   background-color: white;
 }
```

Comment:
- kindly, Could you increase the page color contrast ratio, to at least 4.5:1 for normal-size text? 👍 to have better user interactivity👍👌
- Please, check your color contrast [result](https://www.aditus.io/button-contrast-checker/rawcdn-githack-com-2023-02-01-at-08-35-14-374?overview), and update the [failures](https://www.aditus.io/button-contrast-checker/rawcdn-githack-com-2023-02-01-at-08-35-14-374?failing) 👍
some of them are here with images 👇👇
<img width="1263" alt="Screenshot 2023-02-01 at 11 45 05 AM" src="https://user-images.githubusercontent.com/52010789/215994255-23816435-23e2-42ab-96c6-25024b4fd2f6.png">
<img width="1263" alt="Screenshot 2023-02-01 at 11 44 52 AM" src="https://user-images.githubusercontent.com/52010789/215994263-ad68c036-efb5-44df-9cc1-2874bdb585f0.png">
- Great, Job for implementing the `hover` state, I think it would be better if you implement the `pressed` and `disabled` state based on the given Figma design 👍👍 
<img width="792" alt="Screenshot 2023-02-01 at 11 49 55 AM" src="https://user-images.githubusercontent.com/52010789/215995905-6d2a405b-52d5-4963-886e-25568b20457e.png">